src/IdentityServer/ResponseHandling/Models/InteractionResponse.cs

```diff
@@ -1,16 +1,48 @@
-﻿// Copyright (c) Duende Software. All rights reserved.
+// Copyright (c) Duende Software. All rights reserved.
 // See LICENSE in the project root for license information.
 
 
 using Duende.IdentityServer.Extensions;
 
 namespace Duende.IdentityServer.ResponseHandling;
 
+/// <summary>
+/// Models the types of interaction results from the IAuthorizeInteractionResponseGenerator
+/// </summary>
+public enum InteractionResponseType
```

Comment:
Would it make sense to put Login, Consent, and Create into this enum?

Reply:
The point was to have more of a summary, rather than specific flags to check.

Reply:
Maybe there's a better way to model this, tho?

Reply:
We looked at this together, but didn't like the alternatives - we want to keep existing stuff for backwards compat, and can't cleverly relate the old and new (the collection of booleans might be a weird mixture, and what then?)